src/drawcontext.cpp

```diff
@@ -709,9 +709,6 @@ rvoe<NoReturnValue> PdfDrawContext::convert_to_output_cs_and_set_color(const Dev
 }
 
 rvoe<NoReturnValue> PdfDrawContext::set_color(CapyPDF_PatternId id, bool stroke) {
-    if(context_type != CAPY_DC_PAGE) {
-        RETERR(PatternNotAccepted);
-    }
```

Comment:
FWICR this is here because you are not allowed to use patterns in some drawing contexts. I don't remember what the actual details were. I guess we can remove this and add the more precise check back in later when the issue appears.